### PR TITLE
fix missing command case

### LINKED ISSUE #39683: Test failed: pauseonstart
https://github.com/dotnet/runtime/pull/39129 re-enabled the `pauseonstart` test, which is now failing in every outerloop run.

https://dev.azure.com/dnceng/public/_build/results?buildId=738397&view=ms.vss-test-web.build-test-results-tab&runId=22868214&resultId=110551&paneView=debug

```
    tracing\eventpipe\pauseonstart\pauseonstart\pauseonstart.cmd [FAIL]
      
      cmdLine:C:\h\w\A706091E\w\B72609DC\e\tracing\eventpipe\pauseonstart\pauseonstart\pauseonstart.cmd Timed Out (timeout in milliseconds: 600000 from variable __TestTimeout, start: 7/21/2020 12:16:10 AM, end: 7/21/2020 12:26:10 AM)
      
      Return code:      -100
      Raw output file:      C:\h\w\A706091E\w\B72609DC\e\tracing\eventpipe\Reports\tracing.eventpipe\pauseonstart\pauseonstart\pauseonstart.output.txt
      Raw output:
      BEGIN EXECUTION
       "C:\h\w\A706091E\p\corerun.exe" pauseonstart.dll 
        0.0s: ::== Running test: TEST_RuntimeResumesExecutionWithCommand
        0.1s: Server name is 'DOTNET_TRACE_TESTS_hhvqvdyy.y5u'
        1.0s: running sub-process: C:\h\w\A706091E\p\CoreRun.exe C:\h\w\A706091E\w\B72609DC\e\tracing\eventpipe\pauseonstart\pauseonstart\pauseonstart.dll 0
        1.6s: subprocess started: True
        1.6s: subprocess PID: 3776
        4.4s: { Magic=System.Byte[]; ClrInstanceId=1aaa43bc-df10-84af-ac7f-463e0548be73; ProcessId=3776; Unused=0; }
        4.4s: Sent: { Header={ Magic=System.Byte[]; Size=20; CommandSet=4; CommandId=1; Reserved=0 }; }
        4.5s: received: { Header={ Magic=System.Byte[]; Size=24; CommandSet=255; CommandId=255; Reserved=0 }; Payload=[ 0x85 0x13 0x13 0x80  ]}
        4.5s: Sending 'exit' to subprocess stdin
        9.5s: Subprocess didn't exit in 5 seconds!
       14.5s: Subprocess didn't exit in 5 seconds!
       19.5s: Subprocess didn't exit in 5 seconds!
       24.6s: Subprocess didn't exit in 5 seconds!
       29.6s: Subprocess didn't exit in 5 seconds!
       34.6s: Subprocess didn't exit in 5 seconds!
       39.6s: Subprocess didn't exit in 5 seconds!
       44.6s: Subprocess didn't exit in 5 seconds!
       49.6s: Subprocess didn't exit in 5 seconds!
       54.6s: Subprocess didn't exit in 5 seconds!
       59.6s: Subprocess didn't exit in 5 seconds!
       64.6s: Subprocess didn't exit in 5 seconds!
       69.6s: Subprocess didn't exit in 5 seconds!
       74.6s: Subprocess didn't exit in 5 seconds!
       79.6s: Subprocess didn't exit in 5 seconds!
       84.6s: Subprocess didn't exit in 5 seconds!
       89.6s: Subprocess didn't exit in 5 seconds!
       94.6s: Subprocess didn't exit in 5 seconds!
       99.6s: Subprocess didn't exit in 5 seconds!
      104.6s: Subprocess didn't exit in 5 seconds!
      109.7s: Subprocess didn't exit in 5 seconds!
      114.7s: Subprocess didn't exit in 5 seconds!
      119.7s: Subprocess didn't exit in 5 seconds!
      124.7s: Subprocess didn't exit in 5 seconds!
      129.7s: Subprocess didn't exit in 5 seconds!
      134.7s: Subprocess didn't exit in 5 seconds!
      139.7s: Subprocess didn't exit in 5 seconds!
      144.7s: Subprocess didn't exit in 5 seconds!
      149.7s: Subprocess didn't exit in 5 seconds!
      154.7s: Subprocess didn't exit in 5 seconds!
      159.7s: Subprocess didn't exit in 5 seconds!
      164.7s: Subprocess didn't exit in 5 seconds!
      169.7s: Subprocess didn't exit in 5 seconds!
      174.8s: Subprocess didn't exit in 5 seconds!
      179.8s: Subprocess didn't exit in 5 seconds!
      184.8s: Subprocess didn't exit in 5 seconds!
      189.8s: Subprocess didn't exit in 5 seconds!
      194.8s: Subprocess didn't exit in 5 seconds!
      199.8s: Subprocess didn't exit in 5 seconds!
      204.8s: Subprocess didn't exit in 5 seconds!
      209.8s: Subprocess didn't exit in 5 seconds!
      214.8s: Subprocess didn't exit in 5 seconds!
      219.8s: Subprocess didn't exit in 5 seconds!
      224.8s: Subprocess didn't exit in 5 seconds!
      229.8s: Subprocess didn't exit in 5 seconds!
      234.9s: Subprocess didn't exit in 5 seconds!
      239.9s: Subprocess didn't exit in 5 seconds!
      244.9s: Subprocess didn't exit in 5 seconds!
      249.9s: Subprocess didn't exit in 5 seconds!
      254.9s: Subprocess didn't exit in 5 seconds!
      259.9s: Subprocess didn't exit in 5 seconds!
      264.9s: Subprocess didn't exit in 5 seconds!
      269.9s: Subprocess didn't exit in 5 seconds!
      274.9s: Subprocess didn't exit in 5 seconds!
      279.9s: Subprocess didn't exit in 5 seconds!
      284.9s: Subprocess didn't exit in 5 seconds!
      289.9s: Subprocess didn't exit in 5 seconds!
      295.0s: Subprocess didn't exit in 5 seconds!
      300.0s: Subprocess didn't exit in 5 seconds!
      305.0s: Subprocess didn't exit in 5 seconds!
      310.0s: Subprocess didn't exit in 5 seconds!
      315.0s: Subprocess didn't exit in 5 seconds!
      320.0s: Subprocess didn't exit in 5 seconds!
      325.0s: Subprocess didn't exit in 5 seconds!
      330.0s: Subprocess didn't exit in 5 seconds!
      335.0s: Subprocess didn't exit in 5 seconds!
      340.0s: Subprocess didn't exit in 5 seconds!
      345.0s: Subprocess didn't exit in 5 seconds!
      350.0s: Subprocess didn't exit in 5 seconds!
      355.0s: Subprocess didn't exit in 5 seconds!
      360.0s: Subprocess didn't exit in 5 seconds!
      365.0s: Subprocess didn't exit in 5 seconds!
      370.0s: Subprocess didn't exit in 5 seconds!
      375.1s: Subprocess didn't exit in 5 seconds!
      380.1s: Subprocess didn't exit in 5 seconds!
      385.1s: Subprocess didn't exit in 5 seconds!
      390.1s: Subprocess didn't exit in 5 seconds!
      395.1s: Subprocess didn't exit in 5 seconds!
      400.1s: Subprocess didn't exit in 5 seconds!
      405.1s: Subprocess didn't exit in 5 seconds!
      410.1s: Subprocess didn't exit in 5 seconds!
      415.1s: Subprocess didn't exit in 5 seconds!
      420.1s: Subprocess didn't exit in 5 seconds!
      425.1s: Subprocess didn't exit in 5 seconds!
      430.1s: Subprocess didn't exit in 5 seconds!
      435.1s: Subprocess didn't exit in 5 seconds!
      440.1s: Subprocess didn't exit in 5 seconds!
      445.1s: Subprocess didn't exit in 5 seconds!
      450.1s: Subprocess didn't exit in 5 seconds!
      455.1s: Subprocess didn't exit in 5 seconds!
      460.1s: Subprocess didn't exit in 5 seconds!
      465.1s: Subprocess didn't exit in 5 seconds!
      470.2s: Subprocess didn't exit in 5 seconds!
      475.2s: Subprocess didn't exit in 5 seconds!
      480.2s: Subprocess didn't exit in 5 seconds!
      485.2s: Subprocess didn't exit in 5 seconds!
      490.2s: Subprocess didn't exit in 5 seconds!
      495.2s: Subprocess didn't exit in 5 seconds!
      500.2s: Subprocess didn't exit in 5 seconds!
      505.2s: Subprocess didn't exit in 5 seconds!
      510.2s: Subprocess didn't exit in 5 seconds!
      515.2s: Subprocess didn't exit in 5 seconds!
      520.2s: Subprocess didn't exit in 5 seconds!
      525.3s: Subprocess didn't exit in 5 seconds!
      530.3s: Subprocess didn't exit in 5 seconds!
      535.3s: Subprocess didn't exit in 5 seconds!
      540.3s: Subprocess didn't exit in 5 seconds!
      545.3s: Subprocess didn't exit in 5 seconds!
      550.3s: Subprocess didn't exit in 5 seconds!
      555.3s: Subprocess didn't exit in 5 seconds!
      560.3s: Subprocess didn't exit in 5 seconds!
      565.3s: Subprocess didn't exit in 5 seconds!
      570.3s: Subprocess didn't exit in 5 seconds!
      575.3s: Subprocess didn't exit in 5 seconds!
      580.3s: Subprocess didn't exit in 5 seconds!
      585.3s: Subprocess didn't exit in 5 seconds!
      590.3s: Subprocess didn't exit in 5 seconds!
      595.4s: Subprocess didn't exit in 5 seconds!
      
      cmdLine:C:\h\w\A706091E\w\B72609DC\e\tracing\eventpipe\pauseonstart\pauseonstart\pauseonstart.cmd Timed Out (timeout in milliseconds: 600000 from variable __TestTimeout, start: 7/21/2020 12:16:10 AM, end: 7/21/2020 12:26:10 AM)
      Test Harness Exitcode is : -100
```

@josalem

### PR DESCRIPTION
**Pull request by josalem**
resolves #39683

Since the test was off when it was merged, it didn't catch the missing switch case statement.

### CODE REVIEW

**Review by sywhang — APPROVED** [review]

**Comment by sywhang** [comment]
@josalem can we make sure the failing test (pauseonstart) is running and passing via this PR before checking this in?

**Comment by josalem** [comment]
> @josalem can we make sure the failing test (pauseonstart) is running and passing via this PR before checking this in?

The test is on and should block merging if it fails.  I can remove the auto-merge and merge it in the morning to make sure though.

**Comment by sywhang** [comment]
Thanks!

**Comment by BruceForstall** [comment]
Every outerloop job is going to fail until this is fixed, so I suggest either disabling, then re-enabling with the fix tested, or active monitor and check this in today.

**Comment by sywhang** [comment]
@BruceForstall I can active monitor and merge this in later today if @josalem is not around.

**Comment by josalem** [comment]
Noted.  I’m checking on it actively this evening and will merge when CI is green.

**Pull request merged**
